Dockerfile

```diff
@@ -17,6 +14,14 @@ RUN mv ./kubectl /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
 
 # Add scripts for Istio
 COPY scripts /usr/local/app/scripts/
-RUN chmod +x /usr/local/app/scripts/init_kubeconfig.sh /usr/local/app/scripts/run.sh /usr/local/app/scripts/create_istio_system.sh /usr/local/app/scripts/uninstall_istio_system.sh /usr/local/app/scripts/get_grafana_dashboards.sh
+RUN chmod +x /usr/local/app/scripts/init_kubeconfig.sh /usr/local/app/scripts/run.sh /usr/local/app/scripts/create_istio_system.sh /usr/local/app/scripts/uninstall_istio_system.sh /usr/local/app/scripts/get_grafana_dashboards.sh /usr/local/app/scripts/setup_release_mirror.sh /usr/local/app/scripts/generate_ssl.sh
 RUN mkdir -p /usr/local/app/dashboards && /usr/local/app/scripts/get_grafana_dashboards.sh
+
+# Add nginx configuration
+COPY nginx.conf /etc/nginx/nginx.conf
+
+# Get Istio tar for nginx
+RUN mkdir -p /usr/share/nginx/html/istio/istio/releases/download/${ISTIO_VERSION}
```

Comment:
I think we have to download and add artefacts of all istio versions we want to support upgrades from, and not only the current one.

Reply:
You are correct! I added back in your scrip for pulling in all the images.